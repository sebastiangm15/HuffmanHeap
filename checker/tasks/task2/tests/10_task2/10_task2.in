12
3 FIL83
3 UGD54
10 UGK98
20 GGH93
21 MGW99
40 YFX66
43 RHE34
51 PMD48
58 PPK46
67 RLR10
69 AGY82
96 ZVN12
2
101001101111
0111010111111010101001101100110
