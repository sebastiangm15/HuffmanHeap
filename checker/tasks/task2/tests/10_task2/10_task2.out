RLR10 ZVN12 AGY82 RHE34 
UGK98 GGH93 AGY82 RLR10 PMD48 MGW99 AGY82 MGW99 
